.github/workflows/downstream_quickstarts.yml

```diff
@@ -37,15 +38,15 @@ jobs:
           path: ./timefold-quickstarts
           fetch-depth: 0 # Otherwise merge will fail on account of not having history.
       - name: Checkout timefold-quickstarts (development) # Checkout the development branch if the PR branch does not exist
-        if: steps.checkout-quickstarts.outcome != 'success'
+        if: ${{ steps.checkout-quickstarts-pr.outcome != 'success' }}
         uses: actions/checkout@v4
         with:
           repository: TimefoldAI/timefold-quickstarts
           ref: development
           path: ./timefold-quickstarts
           fetch-depth: 0 # Otherwise merge will fail on account of not having history.
       - name: Prevent stale fork of quickstarts
-        if: steps.checkout-solver-quickstarts.outcome == 'success' # Only run if not on the main line.
+        if: ${{ steps.checkout-solver-quickstarts-pr.outcome == 'success' }}
```

Comment:
I don't see any `checkout-solver-quickstarts-pr` ID in this file.

Reply:
Well spotted!